### ISSUE TITLE
No longer works with JF 10.7.5

### ISSUE THREAD

**Issue by Nazar78** [crobibero/jellyfin-plugin-tmdb-trailers#7]
Since the last JF 10.7.5 update it no longer works.

> [2021-05-15 09:51:28.039 +08:00] [ERR] [72] Jellyfin.Plugin.Tmdb.Trailers.TmdbManager: GetPlaybackUrlAsync
YouTubeFetcher.Core.Exceptions.YouTubeServiceException: There was a problem fetching video information for lEBPNi4bEbc. Not Found
   at YouTubeFetcher.Core.Services.YouTubeService.GetInformationAsync(String id)
   at YouTubeFetcher.Core.Services.YouTubeService.GetStreamingDataAsync(String id)
   at Jellyfin.Plugin.Tmdb.Trailers.TmdbManager.GetPlaybackUrlAsync(String site, String key)
[2021-05-15 09:51:28.041 +08:00] [ERR] [72] Jellyfin.Plugin.Tmdb.Trailers.TmdbManager: GetMediaSource
YouTubeFetcher.Core.Exceptions.YouTubeServiceException: There was a problem fetching video information for lEBPNi4bEbc. Not Found
   at YouTubeFetcher.Core.Services.YouTubeService.GetInformationAsync(String id)
   at YouTubeFetcher.Core.Services.YouTubeService.GetStreamingDataAsync(String id)
   at Jellyfin.Plugin.Tmdb.Trailers.TmdbManager.GetPlaybackUrlAsync(String site, String key)
   at Jellyfin.Plugin.Tmdb.Trailers.TmdbManager.GetMediaSource(String id)
[2021-05-15 09:51:28.043 +08:00] [ERR] [72] Emby.Server.Implementations.Library.MediaSourceManager: Error getting media sources
YouTubeFetcher.Core.Exceptions.YouTubeServiceException: There was a problem fetching video information for lEBPNi4bEbc. Not Found
   at YouTubeFetcher.Core.Services.YouTubeService.GetInformationAsync(String id)
   at YouTubeFetcher.Core.Services.YouTubeService.GetStreamingDataAsync(String id)
   at Jellyfin.Plugin.Tmdb.Trailers.TmdbManager.GetPlaybackUrlAsync(String site, String key)
   at Jellyfin.Plugin.Tmdb.Trailers.TmdbManager.GetMediaSource(String id)
   at Jellyfin.Plugin.Tmdb.Trailers.Channels.TmdbTrailerChannel.GetChannelItemMediaInfo(String id, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Channels.ChannelManager.GetChannelItemMediaSourcesInternal(IRequiresMediaInfoCallback channel, String id, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Channels.ChannelManager.GetDynamicMediaSources(BaseItem item, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.GetDynamicMediaSources(BaseItem item, IMediaSourceProvider provider, CancellationToken cancellationToken)

**Comment by crobibero** [comment]
Thank you for letting me know, I will take a look tomorrow

**Comment by crobibero** [comment]
I can't reliably reproduce this playing the same video that you are (The Marksman Trailer #1 (2021) | Movieclips Trailers) My guess is there's something funky with how the playback url is being pulled

**Comment by Nazar78** [comment]
Hi Cody,

Thanks for your reply. I found it strange that I noticed there's multiple versions of the TMDb Trailers installed, 3 actually. Two of 5.2021.515.1309, one active the other disabled then there's the older previous version marked as supersede. I recalled installing this plugin only once then let the plugin auto update handle the rest. It stopped working after upgrading to JF 10.7.5. Now that I removed all of them then reinstall the latest finally restart, I got it working again. Probably something flaky with the upgrade process. Will monitor and close this for now, thanks again!
